js/react_native/android/src/androidTest/java/ai/onnxruntime/reactnative/OnnxruntimeModuleTest.java

```diff
@@ -149,4 +156,49 @@ public void onnxruntime_module() throws Exception {
       mockSession.finishMocking();
     }
   }
+
+  @Test
+  public void onnxruntime_module_append_nnapi() throws Exception {
+    MockitoSession mockSession = mockitoSession().mockStatic(Arguments.class).startMocking();
+    try {
+      when(Arguments.createMap()).thenAnswer(i -> new JavaOnlyMap());
+      when(Arguments.createArray()).thenAnswer(i -> new JavaOnlyArray());
+
+      OnnxruntimeModule ortModule = new OnnxruntimeModule(reactContext);
+      ortModule.blobModule = blobModule;
+      String sessionKey = "";
+
+      // test loadModel() with nnapi ep options
+      {
+        InputStream modelStream =
```

Comment:
do we need to close the InputStream? consider using [try-with-resources](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html)

Reply:
thanks for the suggestion and catch. added a modelstream.close() here for simplicity.

Reply:
Would using `try` also ensure it's closed if there's an exception?

Reply:
I see. updated to `try-with-resources` block which can ensure the resource will be closed in all cases.